template/web/.env.local.example

```diff
@@ -1,3 +1,6 @@
 NEXT_PUBLIC_GOOGLE_ANALYTICS_ID=
 NEXT_PUBLIC_WALLET_CONNECT_PROJECT_ID="GET_ID_FROM_WALLET_CONNET" # See https://cloud.walletconnect.com
+NEXT_PUBLIC_RPC_URL="GET_FROM_COINBASE_CLOUD"
```

Comment:
I worry this example won't work out-of-the-box for new users.  Is there a way for us to support this at a project level on Sepolia without users needing to create accounts?